Source/WebCore/Modules/WebGPU/GPUQueue.idl

```diff
@@ -44,13 +44,13 @@ interface GPUQueue {
     undefined writeBuffer(
         GPUBuffer buffer,
         GPUSize64 bufferOffset,
-        [AllowShared] BufferSource data,
+        [AllowShared] (ArrayBufferView or ArrayBuffer) data,
```

Comment:
In `WebGLRenderingContextBase.idl`, I see:
```
typedef ([AllowShared] ArrayBuffer or [AllowShared] ArrayBufferView) AllowSharedBufferDataSource;
typedef ([AllowShared] Float32Array or sequence<GLfloat>) Float32List;
```
... where the parentheses goes around the whole thing.  Is it OK that your parentheses goes around the 2 variant types?

Reply:
Yes, this is fine, attaching `[AllowShared]` to `(ArrayBufferView or ArrayBuffer)` union.